README.md

```diff
@@ -105,6 +105,25 @@ This id is injected in req object, then if you need to log some extra informatio
 logger.info('some controller information', { id: req.id })
 ```
 
+## Examples
+
+The log-generator inside examples folder will run a nodejs application that will make request for itself every milliseconds decided by the user. The application will get the milleseconds value from an environment variable called `ESCRIBA_TIMEOUT`.
+
+To use log-generator through Docker use this commands:
+
+```
+docker build -t pagarme/log-generator:latest .
+docker run -e ESCRIBA_TIMEOUT=5000 -p 3000:3000 -d pagarme/log-generator:latest
+```
+
```

Comment:
Another good information to the user that do not masters nodejs, is how to use the library inside the container bypassing the version installed from `npm install`, something like: `docker run -d -v $(cd ../../ && pwd):/escriba  pagarme/log-generator:latest`